upi/azure/azuredeploy.json

```diff
@@ -0,0 +1,1072 @@
+{
+  "$schema" : "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion" : "1.0.0.0",
+  "parameters" : {
+    "image" : {
+      "type" : "string",
+      "metadata" : {
+        "description" : "OS to use.Red Hat Enterprise Linux"
+      }
+    },
+    "BootstrapIgnition" : {
```

Comment:
is it possible for it to load the file from disk?

Reply:
Azure for RHCOS uses a "custom" version of cloud init. Max size is quite limited. In RHEL there is a ability to pass a file, but it relies on a very agent that has its own interesting issues. While master/nodes can directly be passed there ignition file, I did test, and found that the bootstrap ignition is about 10x to big for azure cloudinit. I do know there a long standing request for microsoft/azure to enhance this, but its been ongoing for a 18months plus, and no end in sight. Right now bootstrap ignition is copied to a storage account in the resource group, and deleted on bootstrap complete. We can further harden this a few ways. One is move the copy to part 2, which means it exists "less" time, second is a bit more intense is see if it can be limited to access from within the resource group. The first part is easy, the second part is 3 days.